### ISSUE TITLE
Host System monitoring NETTX NETRX

### ISSUE THREAD

**Issue by feldsam** [OpenNebula/one#6114]
**Description**
NETTX and NETRX in new host monitoring are None/Zero

**Details**
 - Affected Component: [Monitord]
 - Hypervisor: [KVM]
 - Version: [6.2]

**Additional context**
Looks like something happens in new monitoring system.  When I check directly on node, linux_host.rb returns NETTX and NETRX
```
[root@dc3-node1 monitor]# ./linux_usage.rb 
HYPERVISOR=kvm
USEDMEMORY=243410764
FREEMEMORY=20606980
FREECPU=3888
USEDCPU=912
NETRX=41307203246698
NETTX=50104108822222
```

## Progress Status
- [ ] Code committed
- [ ] Testing - QA
- [ ] Documentation (Release notes - resolved issues, compatibility, known issues)

**Comment by paczerny** [comment]
Duplicate #5640, fixed in 6.6.0 and 6.4.3 release

**Comment by feldsam** [comment]
Hi @paczerny, I don't think it was solved in #5640 - it was about VM not HOST. I applied path to 6.2 release and run new VM on host id 2. In host_monitoring db table is not NETTX or NETRX

```
<MONITORING><TIMESTAMP>1675762479</TIMESTAMP><ID>1</ID><CAPACITY><FREE_CPU><![CDATA[3792]]></FREE_CPU><FREE_MEMORY><![CDATA[20504788]]></FREE_MEMORY><USED_CPU><![CDATA[1008]]></USED_CPU><USED_MEMORY><![CDATA[242982228]]></USED_MEMORY></CAPACITY><SYSTEM></SYSTEM></MONITORING>
```

**Comment by paczerny** [comment]
Hi @feldsam,
sorry, I haven't notice the difference to previous issue. I'm reopening this.
